src/find-with-ref.js

```diff
@@ -24,3 +24,10 @@ export default function findWithRef(tree, ref) {
 
   return found[0];
 }
+
+export default function findWithRef(root, ...refs) {
+  if(refs.length == 1)
+    return findWithRefForOne(root, refs[0]);
+
+  return refs.map(ref => findWithRefForOne(root, ref));
```

Comment:
We could optimize this by changing the `findAll` function to take in multiple options so that it does the tree traversal only once. But if we need this just for `findWithRef` those changes would seem unnecessary